tests/02-compile-arm-ports/Makefile

```diff
@@ -45,6 +45,7 @@ libs/shell/openmote \
 libs/simple-energest/openmote \
 libs/timers/zoul \
 libs/trickle-library/zoul \
+libs/unit-tests/zoul \
```

Comment:
I think it would be much better to add execution test rather than just a compilation test. A new test could be added in 07-simulation-base, and then this change removed.

Reply:
No new commits, but I think we should merge anyway.